### ISSUE TITLE
Using DeepChem with a newer RDkit

### ISSUE THREAD

**Issue by HenriqueCSJ** [deepchem/deepchem#1579]
Dear colleagues,
I'm using Deepchem 2.1.0 Python 3.6 and my work is dealing with rather complex organometallic structures.

As I can see DeepChem is using RDKit 2017.09.01 and crashes when importing my dataset while RDKir 2019.03.01 (used alone) works fine. My question is: is there a way to use a new RDKit version with DeepChem? Do you have any plans to upgrade?

Thanks in advance

**Comment by peastman** [comment]
Agreed, we really should update the install script to specify a more recent RDKit.  Once you have DeepChem installed, you should be able to just `conda install` a newer version and it should work.

**Comment by mrshirts** [comment]
Yeah, I'm having the same issue; I can't even conda install rdkit=2017.09.01 from conda-forge - it's not clear that that particular release is on conda-forge or rdkit anymore.  conda install -c rdkit rdkit=2017.09.1 is stalling for the last couple of hours, but it's definitely not available on conda-forge

Was going to demonstrate molecular featurization in deepchem with my course tomorrow; we'll see if I can get something working . . .

**Comment by peastman** [comment]
I have RDKit 2018.09.2.0 installed on my computer and it's been working fine with DeepChem.

**Comment by mrshirts** [comment]
OK, working on figuring out how to get it to install despite the conda warning on

`conda install deepchem`

`PackagesNotFoundError: The following packages are not available from current channels:
  - deepchem -> rdkit==2017.09.1`

And use the later rdkit version.  Haven't dug into conda quite that before.

**Comment by peastman** [comment]
I installed DeepChem from source, so that wasn't an issue.  @lilleswing we *really* need to get the DeepChem 2.2 conda build out ASAP, and have it specify an up to date RDKit version.

**Comment by mrshirts** [comment]
yeah, I can't have the students do that :)  Many of them are on Windows.  I may try to get it running on my computer from source to at least demo.

**Comment by HenriqueCSJ** [comment]
> 
> 
> I installed DeepChem from source, so that wasn't an issue. @lilleswing we _really_ need to get the DeepChem 2.2 conda build out ASAP, and have it specify an up to date RDKit version.

Thanks for taking a look at this!
I'd really like to use DeepChem on my thesis instead of runing a bunch of different code.

**Comment by VoVAllen** [comment]
Same problem here

**Comment by HenriqueCSJ** [comment]
Please, take a look at this issue. The newer DeepChem version is still using RDKit 2017.
Ironically, DeepChem 1.1 IS using the new RDKit.
It makes a huge difference in how RDKit handles coordination compounds (or at all).
```
The following packages will be UPDATED:

  deepchem                          omnia::deepchem-1.1.0-0 --> deepchem::deepchem-2.1.0-py36_0

The following packages will be SUPERSEDED by a higher-priority channel:

  cairo              conda-forge::cairo-1.16.0-h18b612c_10~ --> pkgs/main::cairo-1.14.12-h7636065_2
  grpcio             pkgs/main::grpcio-1.16.1-py36hf8bcb03~ --> conda-forge::grpcio-1.16.0-py36h4f00d22_1000
  joblib             conda-forge/noarch::joblib-0.13.2-py_0 --> conda-forge/linux-64::joblib-0.11-py36_0
  pillow             conda-forge::pillow-5.4.1-py36h00a061~ --> pkgs/main::pillow-5.0.0-py36h3deb7b8_0
  scikit-learn       conda-forge::scikit-learn-0.21.3-py36~ --> pkgs/main::scikit-learn-0.19.1-py36hedc7406_0
  tensorflow         pkgs/main::tensorflow-1.14.0-mkl_py36~ --> conda-forge::tensorflow-1.6.0-py36_0

The following packages will be DOWNGRADED:

  fontconfig                           2.13.1-he4413a7_1000 --> 2.13.0-hd36ec8e_5
  freetype                                2.10.0-he983fc9_0 --> 2.8.1-hfa320df_1
  glib                                 2.58.3-h6f030ca_1002 --> 2.56.2-had28632_1001
  libblas                                 3.8.0-11_openblas --> 3.8.0-6_h6e990d7_netlib
  libcblas                                3.8.0-11_openblas --> 3.8.0-6_h6e990d7_netlib
  liblapack                               3.8.0-11_openblas --> 3.8.0-6_h6e990d7_netlib
  networkx                                         2.3-py_0 --> 2.1-py_1
  openssl                                 1.1.1c-h516909a_0 --> 1.0.2r-h14c3975_0
  pandas                              0.25.0-py36hb3f55d8_0 --> 0.22.0-py36_1
  pdbfixer                                       1.5-py36_0 --> 1.4-py36_0
  python                                3.6.7-h357f687_1005 --> 3.6.7-hd21baee_1002
  rdkit                          2019.03.3.0-py36hc20afe1_1 --> 2017.09.1-py36_1
  readline                                   8.0-hf8c457e_0 --> 7.0-hf8c457e_1001
  sqlite                                  3.29.0-hcee41ef_0 --> 3.28.0-h8b20d00_0
  tensorboard                                 1.14.0-py36_0 --> 1.6.0-py36_0

```

**Comment by peastman** [comment]
This is supposed to be fixed in 2.2.0, but something strange is going on with conda.  When I tell it to install `deepchem=2.2.0`, the only package it finds is the Python 2.7 one.  I don't understand why, because the Python 3.6 and 3.7 packages are also listed at https://anaconda.org/deepchem/deepchem/files.  @lilleswing do you have any idea about this?

**Comment by nd-02110114** [comment]
It seems this issue was solved. I closed this issue
